app/scripts/services/api-explorer.js

```diff
@@ -626,6 +626,21 @@
                     });
                     return deferred.promise;
                 },
+
```

Comment:
Seeing as this is a new portable service method that introduces new functionality, please document in a header on the function what it does and what the expected return values are.

Reply:
Added the header